packages/plugin-assets-retry/src/runtime/initialChunkRetry.ts

```diff
@@ -75,9 +76,18 @@ function validateTargetInfo(
   ) {
     return false;
   }
+
   return { target, tagName, url };
 }
 
+const QUERY_REG = /\?.*$/;
+function removeQuery(url: string) {
+  return url.replace(QUERY_REG, '');
+}
+function getQueryFromUrl(url: string) {
```

Comment:
Can we move the duplicated helpers to a shared module?

Reply:
maybe in next pr...
we need to migrate build script from ("babel" + "terser") to "modernjs-module-tools"
I'm not sure what unknown problems will occur.